frontend/cypress/integration/multiPart.spec.js

```diff
@@ -0,0 +1,296 @@
+import faker from 'faker'
+const syncedRepoUrlMultiParts =
+  'https://github.com/kitspace-forks/DIY_particle_detector'
+const syncedRepoUrl = 'https://github.com/kitspace-forks/CH330_Hardware'
+const multiPartsNames = ['alpha-spectrometer', 'electron-detector']
+const multiPartsRepoName = syncedRepoUrlMultiParts.split('/').slice(-1).toString()
+const normalRepoName = 'CH330_Hardware'
+
```

Comment:
Again, I think all of these should be called `multiProject...`.

Reply:
Resolved in 2fd171f